.github/workflows/build-and-artifact.yml

```
@@ -0,0 +1,19 @@
+name: Build and Artifact
```

actionlint (1.7.4)

"on" section is missing in workflow

(syntax-check)